src/support.njk

```diff
@@ -81,20 +81,4 @@ hubspot:
       </div>
     </div>
   </div>
-
-<link rel="preload"href="https://cdn.jsdelivr.net/npm/@algolia/algoliasearch-netlify-frontend@1/dist/algoliasearchNetlify.css" as="style" onload="this.onload=null;this.rel='stylesheet'" crossorigin="anonymous" referrerpolicy="no-referrer" />
-<link rel="preload" href="https://cdn.jsdelivr.net/npm/@algolia/autocomplete-theme-classic" as="style" onload="this.onload=null;this.rel='stylesheet'" crossorigin="anonymous" referrerpolicy="no-referrer" />
-
-<script async type="text/javascript" src="https://cdn.jsdelivr.net/npm/@algolia/algoliasearch-netlify-frontend@1/dist/algoliasearchNetlify.js" onload="initSearchBar()"></script>
-<script async type="text/javascript">
-  function initSearchBar () {
-    algoliasearchNetlify({
-        appId: 'KWE0727TZS',
-        apiKey: '595ee07f069991380e9c3036cfad8e5e',
-        siteId: '00f8cf60-997f-4c4d-9427-a97924358648',
-        branch: 'live',
-        selector: 'div#algolia-search',
-        detached: false
-    });
-    }
-</script>
```

Comment:
@cstns, I think we should include the `common-js.njk` file here, right?

It currently looks like this:
<img width="553" alt="Screenshot 2024-10-09 at 14 35 02" src="https://github.com/user-attachments/assets/48441e7f-4fc4-466b-a522-24507ae59934">

Or did you intend to remove it entirely? It made sense when it was a global search.

Reply:
I'm looking into it, I can't tell right now why it's not working on the deployed branch and working just fine locally.

Also, should this search bar be scoped?

Reply:
nevermind, I was toggling between branches.